project/REVIEWING.md

```diff
@@ -207,3 +197,22 @@ review session. The goal of that session is to agree on one of the following out
  * Escalate to Solomon by formulating a few specific questions on which his answers will allow
    maintainers to decide.
 
+## Milestones
+
+Typically, every merged pull request get shipped naturally with the next release cut from the
+`master` branch (either the next minor or major version). However, the time-based nature of the
```

Comment:
Do we want to point out how / where to find out what the next minor or major version is expected to be?